src/assets/svg/calIcon.svg

```diff
@@ -0,0 +1,9 @@
Mg587d+4xBD91s0BFZG1eWIatNxjtqq2tddTX168IisPGul8bF+FesyEV4IMVJseKnbb0OqP3d9kB4mhV1LXjYi5784Hk6bFRqtMo1qwLRwlwu1J4ffPA8j8qlY6NPYj+vcyYjgluoCis4VQ2J6OgPVByc3M/DefBffLJJ3k0/bIEQJYDhS+m7N577/3q5ptvfgooUUYkgAYgsd9xxx277r777oMAOta5Ipvz70NsErOGpAQ4xvpDhw6ln3322bO/+uqrJTabzdHWMYEy6UGptr700kvpnL/IPovzt3zx5efIWGRz5+Tk/BjshG+++YYVvjD6E5FttuTsEIOg1W1+s+eksUOiPukSo+ov9+INZq9+Z5pjzbs/mf679Ygjk5NYfQ2QvgDmmfClf4FvHiN1Eq7yvvODKQNzfOiBIOgzyfqz9GbZvgcoZxegNPNTU1PvCnXu2rVrS8AK76aXfITi9YXYQ4eeVxQpPTqyA8E6XjJr1qwpQI8GYyOocF4QdpX78ssv89588818sIBmAgCjGHmcvylWMJrBUVpEDPlQ/YcMGTISLPetEydOvCA5OTkunDFhKsK8efNyYFwllM/TQLMR66LH/C6HnBVmemZxQHOORUdHnx74d6PR6AQfYTXM0iWCNZJCgbMvS9qyRZJ67j1JvW6+JO7ZoanaJ8J5WDYnbwU/YmdeuXvF/DXm1ZsO2mvIIgXunXsGdm8Di48LPpPhayXLDcvqPGYAf9aHK33ZnaznThZZQwb+sHoRgQJE9e7du++mTZvmnXXWWZJJWL/++mspWPZ9YEHrSdkOUaSpEgtRhEaD4v09aH2hX0pKSv8HHnhg7Pjx4wcOGjSoOyhbEjidOnjpWsZzcQUVyymPHTvW8OOPP1Zs3LixGiNGgtBfBXHrIqIYxjCApiYlQN8Mw7D9EXB33XXX2Msvv3wozA59+vbtm4TdqTGXybc44XR6q6urrQUFBea9e/fWAtcvh59s/2MTGZ9yGlMJBSCMXJj1zxgNmj179lmg8GeXlpaeZbVaT4OZzocBi8XigRm6lr7jCD3vCir84U6EAvhe6NdPp1w49YLYT1IS1CPCnWpgNigFGrXL5eGPROtUBXDoYTbR6DSqbjotN0CtUp0Lj3yCKkTxDUZ7APyZn69tyutn0y6z/CauFY24QAHwheMiXr833njjNnDMbgOQDoyLi4vCxraHDx+uB4uXD/ShhL6jlGacNHrxRmHLD1ICtLqxNBug5cVVV4zEdKPfoQMdrdPposGSasinYM6rsHzRRFa2isDGmm1ZWgE0Nc1Q8TSOXqQMwv6csaAYMQBADaVq84IxOYiKGAVrD+UE/KZ25OHmFtG4kshYDKXZM4HzZxOYOX//Ul8nPHzeJ0oBfBjp31OT+OXslHsuOTv6DQBwfGt5l8PFg7XkjGo1p9NqVD3UQay9UA7kOmvnrTTnLNnkc3iFnD+vLeAnBWBWG512zOXHMryBnD99QUsAsHL+Fipsp5gGTjoXXU3XjaUXmiw4EunaMQRK5pN5AsBv4PxF38K8Gg/XOieQKWcMAZ51mO5KY0qg8UbRebxAIRn42b4PrP29ReALtXZM0fT9PUgZYgQKwHL/2V7HYTvArVoHEIariqs9lm+3WNd1S9KMHjVQdycAV9MaBQDl6U3WUK7CeLYddVS+/YMpGygUK2dkAMwjK9Rq8LPnI3BiS+hlG0kBkgjEbnr41WSNKjnpprmcwJozR5k5cbG0dhBLL52ljgTOACyVmDUasLcR+MJ7dXP+DE0TPVPhuFgxu3BMLoFiCpsfuCM0JqcA3NUB+GRtGR1cBGvNteE+tIUbLGU6reqz2Tcl9AN/4EruOAs6u6v32EvfWm7MpkL2arL8wvweMxeZ/qM85+996eT8iXSsU5lHYJWNAqvHy7guL6IIOs6fOKgOABsrLnIHACySKeZM6d10X1rBEWxMrgiBXspYOLnmGcWB6fUR+05tKx6YHfh3FvD1t5+4MaHLGX20Y+HfmkgDHzM6AfCGBest+eDs5nl5n2WoEoCfLXpZuMg232KgMBEoGjh/RwMhFXC3crrnOX/VnJNrHyJUUGc7GAvHnaBqwrA3yiaNxCm76/RL48575taEp4f3040DhzY2IiaA5/gavce+6ZDD17AWeH+NIPpRSEcZJ+gAHAnwB1aEKdL5JCI7xQscO+Su3Qb01A5+5c7EuyedFzM1JUHdFfh9qzJAXWDxjVbetS/bWTN/jTl/7V5bhcfblOJbRs5mMc0CbBXQ2072HO6wEknFP3GN707uDCBUArT6XWOjVAOuHhMzbsZlcZMvGxU9pmu8OjFKq9JoNdIrzfi1bg/vdbo5b53Ra9+V4aj+30ZL8fY0R43Z1rQhAwuxldAM0NRjUwG+ogAnWwGYEsRQlKRPjyT1GTAjDAElOOf8IVGDwT/o0burJjGpizpap+E0WDSPER2DxessqfFYMksaN9PeesRRU2PwWuD3wh3+Kgn0LJ/cRFEQjwJ+RQFOlhMs5rWz/b+csdEqe3KC2tk9UR2fEKtO1mpUGOOOBUuvwwY0qABAa7x4wL/d8LyccJjhqIFf1HD+6qEqiu6wpCe2y4xXgawi7WkGEDrGbGEFF1FwQQUXM3rRT7bzR4xA6VjozUgOLVOAOs6/sbawgati9Tugk9nZZ4Cme+f8q6RstZCtYGIEJyVAAdj5doEC1HH+FhcK8BXpUDOA2GzAlv+7cP7lfh3nj6ezRQ+WWyLcUcajAF+ZATqqAjAlYI1hhauKmoBZo6lXJHcSWioqoijA8VIARRQF6Bw+QEe8KUUUkStq5REooiiAIoooCqCIIooCKKKIogCKKKIogCKKKAqgiCKKAiiiiKIAiiiiKIAiiigKoIgiigIoooiiAIoooiiAIoooCqCIIooCKKKIogCKKKIogCKKKAqgiCKKAiiiiKIAiiiiKIAiiigKoIgiigIooki7kf8XYABekykBMgkdogAAAABJRU5ErkJggg=="/>
```

Comment:
This should really just be a png file instead of an svg with an embedded image. Switching to png will also greatly reduce the file size.